javascript/first-js-oop-project.md

```diff
@@ -71,10 +75,22 @@ This mini project will help you to learn basic concepts about objects in JavaScr
    // The below statement creates an object.
    const event_obj1 = new Event("KLOS Golden Gala", "An evening with hollywood vampires");
    ```
+1. Now add a function on the class called `addAvailableTickets` to create a ticket type, that will accept two parameters. The method should look like this when it is called:
+```javascript
+   event_obj1.addAvailableTickets("human", 299);
+   event_obj1.addAvailableTickets("vampire", 99);
+```
+\**Hint: You might want to create another class, "Ticket" with two properties on it
```

Comment:
Since they've never made an object before, maybe walk them through how to make the Ticket class and how to make a function on the Event class. Then if you want them to make their own function, you could add another exercise that has them make a function totally independently.